### PR TITLE
{tools}[foss/2022b] snakemake v7.32.3, wrapt v1.15.0

### PR DESCRIPTION
**Pull request by PetrKralCZ**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by PetrKralCZ** [comment]
Test report by @PetrKralCZ
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node4001.donphan.os - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Gold 6240 CPU @ 2.60GHz, 1 x NVIDIA NVIDIA A2, 530.30.02, Python 3.6.8
See https://gist.github.com/PetrKralCZ/c2774e1d4b5c4382aa24abef6e0ec453 for a full test report.

**Comment by PetrKralCZ** [comment]
@boegelbot please test @ generoso

**Comment by casparvl** [comment]
@boegelbot please test @ jsc-zen2

**Comment by casparvl** [comment]
@boegelbot please test @ jsc-zen2

File: easybuild/easyconfigs/s/snakemake/snakemake-7.32.3-foss-2022b.eb [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+    ('plac', '1.3.5', {
+        'checksums': ['38bdd864d0450fb748193aa817b9c458a8f5319fbf97b2261151cfc0a5812090'],
+    }),
+    ('yte', '1.5.1', {
+        'checksums': ['6d0b315b78af83276d78f5f67c107c84238f772a76d74f4fc77905b46f3731f5'],
+    }),
+    ('humanfriendly', '10.0', {
+        'checksums': ['6b0b831ce8f15f7300721aa49829fc4e83921a9a301cc7f606be6686a2288ddc'],
+    }),
+    ('nbformat', '5.9.2', {
```
Comment (casparvl) [inline easybuild/easyconfigs/s/snakemake/snakemake-7.32.3-foss-2022b.eb:66]: I see that `nbformat`, `jupyter-core` and `fastjsonschema` are new (they weren't  in `snakemake-7.18.2-foss-2021b.eb`). I'm wondering if we should try to allign the versions of these with those used in e.g. `jupyter-server`. In the current PR, the versions are slightly different than what was used for `jupyter-server` in the `2022b` toolchain. 

Maybe @boegel has an opinion on that?

Comment (boegel) [reply easybuild/easyconfigs/s/snakemake/snakemake-7.32.3-foss-2022b.eb:66]: I don't think we can reasonable expect to align extension versions across easyconfigs using the same toolchain...

Comment (casparvl) [reply easybuild/easyconfigs/s/snakemake/snakemake-7.32.3-foss-2022b.eb:66]: Ok, in that case: all good to go with this PR! :)

**Review by casparvl — APPROVED** [review]

**Pull request merged**